### PR TITLE
Enhancement/control css ac

### PR DESCRIPTION
**Pull request by Alexc73**

### CODE REVIEW

File: src/components/Toggle/Toggle.js
Comment (CodeDamien1) [inline src/components/Toggle/Toggle.js:1]: what will this toggle function do?

Comment (Alexc73) [reply src/components/Toggle/Toggle.js:1]: switch between Register and Login page

Comment (CodeDamien1) [reply src/components/Toggle/Toggle.js:1]: so basically it allows you to decide wether you want to login if you have an account or decide to register if you dont. sounds good to me please merge this

Comment (Alexc73) [reply src/components/Toggle/Toggle.js:1]: yes

**Pull request merged**